distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/org.eclipse.jdt.core.compiler,ecj,4.6.1.redhat-1,Eclipse Public License 1.0.txt~[KEYCLOAK-16723] Upgrade to Wildfly 22

```diff
@@ -0,0 +1,210 @@
+Eclipse Public License - v 1.0
```

Comment:
Is this change related?

Reply:
@pedroigor I think the file name is not correct on your side ?

Reply:
Please remove changes to license files as they are not related to the Apple Identity Provider (these files are also generated/downloaded, not manually maintained).